targets/targets.json

```diff
@@ -3033,6 +3033,48 @@
         ],
         "device_name": "STM32L072CZ"
     },
+    "STM32L071CXCTX": {
```

Comment:
I think, you don't really add a new target?
So please, renam to MCU_STM32L071xx (like MCU_STM32F103xB for ex)
Then in your custom_target.json, you will inherit from it.

Reply:
ok, got it